frontend/src/scenes/billing/UnsubscribeSurveyModal.tsx

```diff
@@ -65,6 +69,9 @@ export const UnsubscribeSurveyModal = ({ product }: { product: BillingProductV2T
                             status="muted"
                             onClick={() => {
                                 setSurveyID('')
+                                posthog.capture('survey dismissed', {
+                                    $survey_id: surveyID,
+                                })
```

Comment:
oh technically since this is repeated better to define it once and call it from there in the multiple spots..... since it's only twice you can flex a bit on DRY but might be a good quick improvement.

Reply:
Good point, I'll move it into a kea listener.